src/main/java/org/gridsuite/explore/server/services/CaseService.java

```diff
@@ -0,0 +1,84 @@
+package org.gridsuite.explore.server.services;
+
+import org.gridsuite.explore.server.ExploreException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.client.MultipartBodyBuilder;
+import org.springframework.http.codec.multipart.FilePart;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.ClientResponse;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.UriComponentsBuilder;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.util.UUID;
+import java.util.logging.Level;
+
+@Service
+public class CaseService implements IDirectoryElementsService {
+    private static final String ROOT_CATEGORY_REACTOR = "reactor.";
+
+    private static final String CASE_SERVER_API_VERSION = "v1";
+
+    private static final String DELIMITER = "/";
+
+    private final WebClient webClient;
+
+    private String caseServerBaseUri;
+
+    public void setBaseUri(String actionsServerBaseUri) {
+        this.caseServerBaseUri = actionsServerBaseUri;
+    }
+
+    @Autowired
+    public CaseService(@Value("${backing-services.case-server.base-uri:http://case-server/}") String studyServerBaseUri,
```

Comment:
An entry is missing in src/main/resources/application-local.yml, I think

Reply:
done